src/ui/Analytics/AnalyticsActionListMeta.ts

```diff
@@ -1214,5 +1248,55 @@ export var analyticsActionCauseList = {
   foldingShowLess: <IAnalyticsActionCause>{
     name: 'showLessFoldedResults',
     type: 'folding'
+  },
+  /**
+   * Identifies the search event that gets logged when a mLFacet check box is selected and the query is updated.
```

Comment:
Identifies the -> The
mlFacet -> DynamicFacet (that's the new name, right?)

Reply:
This comment also applies to the `mlFacetDeselect` and `mlFacetClearAll` documentation

Reply:
yes but i will change it in another PR